src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
+$card-width: 200px;
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$secondary-color: #616070;
+$button-bg-color: #00acdc;
+$button-bg-color-hover: #fff;
+$button-text-color: white;
+
 body {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
 }
+
+.product-card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  &__img {
+    width: 160px;
+    display: block;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
```

Comment:
Ensure that the image paths used in 'background-image' are correct and that the images are accessible at the given paths in the project structure.